source/kubernetes/create-app/index.html.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Create a new application on the GOV.UK Kubernetes Platform
+weight: 20
+layout: multipage_layout
+---
+
+# Create a new application on the GOV.UK Kubernetes Platform
+
+## Assumptions
+
+This guide assumes the following:
+
+- You have completed all the steps in the ["get started" guide](/manual/get-started.html) of the
+  developer docs
+- You are deploying an application named `example` (make sure to replace this with your
+  application name in all steps as needed)
+- You have generated an initial application and pushed it to a Github repository under the
+  `alphagov` organisation with the `govuk` and `container` labels applied (this triggers setup of a
+  number of Github Actions secrets from the organisation and permissions required for workflows)
+- You have set up the application in [GOV.UK Docker](https://github.com/alphagov/govuk-docker) (not
+  covered by this guide) so that others can run it locally (this will also help you understand what
+  environment configuration your application needs, especially if it is an existing application
+  rather than a brand new one)
+- You have a Sentry project configured for your application (through `govuk-infrastructure`)
+
+## Add Github Actions workflows
+
+### Add CI workflow
+
+What your Continuous Integration (CI) workflow looks like will depend on the linting and testing
+needs of your application, but will act as a dependency for further workflows to block deployment in
+case of failure. A Rails application would usually run RSpec unit and integration/system tests and
+Rubocop.
+
+This workflow should have a name of `CI`. If you're not sure what to include in this workflow, find
+an existing GOV.UK application similar to yours and use its configuration as a starting point (by
+convention, the workflow should be in `.github/workflows/ci.yml`).
+
+### Add release workflow
+
+Add a workflow invoking the shared release workflow template to `.github/workflows/release.yml`, for
+example:
+
+```yml
+name: Release
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
+
+jobs:
+  release:
+    if: github.event_name == 'workflow_dispatch' || github.event.workflow_run.conclusion == 'success'
+    name: Release
+    uses: alphagov/govuk-infrastructure/.github/workflows/release.yml@main
+    secrets:
+      GH_TOKEN: ${{ secrets.GOVUK_CI_GITHUB_API_TOKEN }}
+```
+
+> **Note**
+> You may need to manually create a `v0` tag (e.g. against the most recent commit) in your
+> repository for the `release` workflow to pick up on and start automatically creating releases with
+> incremented versions.
```

Comment:
We need to be specific that they need to create a GitHub release and not just a git tag.
```suggestion
> You may need to manually create a `v0` GitHub release (e.g. against the most recent commit) in your
> repository for the `release` workflow to pick up on and start automatically creating releases with
> incremented versions.
```

Reply:
Interesting - I've definitely only created a `v0` tag rather than a release, and then relied on the workflow to create the `v1` release and tag based on that. The only reason this is really necessary (I think) is because the workflow fails to run if there isn't an initial tag around to increment 😕